src/compiler/visitorPublic.ts

```diff
@@ -624,909 +614,581 @@ type VisitEachChildFunction<T extends Node> = (node: T, visitor: Visitor, contex
 //  }
 //
 // This is then used as the expected type for `visitEachChildTable`.
-type VisitEachChildTable = { [TNode in HasChildren as TNode["kind"]]: VisitEachChildFunction<TNode> };
+type VisitEachChildTable = { [TNode in HasChildren as TNode["kind"]]: VisitEachChildFunction<TNode>; };
 
 // NOTE: Before you can add a new method to `visitEachChildTable`, you must first ensure the `Node` subtype you
 //       wish to add is defined in the `HasChildren` union in types.ts.
 const visitEachChildTable: VisitEachChildTable = {
     [SyntaxKind.QualifiedName]: function visitEachChildOfQualifiedName(node, visitor, context, _nodesVisitor, nodeVisitor, _tokenVisitor) {
-        return context.factory.updateQualifiedName(node,
-            Debug.checkDefined(nodeVisitor(node.left, visitor, isEntityName)),
-            Debug.checkDefined(nodeVisitor(node.right, visitor, isIdentifier)));
+        return context.factory.updateQualifiedName(node, Debug.checkDefined(nodeVisitor(node.left, visitor, isEntityName)), Debug.checkDefined(nodeVisitor(node.right, visitor, isIdentifier)));

```

Comment:
This I previously mentioned, but this section is really bad just from every one of these having the first parameter on the first line making dprint think that this needs to be yanked onto one line.